Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm

```diff
@@ -248,7 +248,7 @@ static inline uint8_t authDataFlags(ClientDataType type, LocalConnection::UserVe
     }
 
     // Step 3.
-    auto existingCredentials = getExistingCredentials(creationOptions.rp.id);
+    auto existingCredentials = getExistingCredentials(*creationOptions.rp.id);
```

Comment:
Since `options.rp.id` may now be `std::nullopt`, do we need to do any validation here? Maybe at least ASSERT so we catch cases where a null opt got through?